l10n_it_financial_statement_eu/__init__.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2023 Associazione Odoo Italia (<http://www.odoo-italia.org>)
+# Copyright 2023 MKT Srl (<www.mkt.it>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import models
+from . import wizard
+from . import report
+
+from odoo import api, SUPERUSER_ID
+
+
+def _l10n_it_financial_statement_eu_post_init(cr, registry):
```

Comment:
Gli **hooks** andrebbero in un file a parte come descritto in https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#installation-hooks